triagebot.toml

```diff
@@ -402,6 +402,10 @@ message_on_add = """\
 Issue #{number} "{title}" has been added.
 """
 
+[no-merges]
+exclude_labels = ["rollup", "subtree-sync"]
```

Comment:
We also want to add those labels to the `no-merges` policy in the Clippy repo: https://github.com/rust-lang/rust-clippy/blob/8c48b936ccf5cf62647420e604bb2cae155268a4/triagebot.toml#L12-L13

Reply:
Ah, we should probably do the same in Miri. And that that point the sync PRs for the rustc → Miri direction (which are auto-created via our cronjob) should then also get that label.